modules/api/src/test/java/org/apache/ignite/network/NodeMetadataTest.java

```diff
@@ -15,8 +15,28 @@
  * limitations under the License.
  */
 
-/**
- * Configuration schemas for Rest component.
- */
+package org.apache.ignite.network;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.nio.ByteBuffer;
+import org.junit.jupiter.api.Test;
+
+class NodeMetadataTest {
+
+    @Test
+    void testToByteBufferAndFromByteBuffer() {
+        NodeMetadata metadata = new NodeMetadata(10000);
+        ByteBuffer buffer = metadata.toByteBuffer();
+        NodeMetadata fromByteBuffer = NodeMetadata.fromByteBuffer(buffer);
+        assertEquals(metadata, fromByteBuffer);
+    }
 
-package org.apache.ignite.internal.rest.configuration;
+    @Test
+    void testFromByteBufferWithWrongContent() {
+        ByteBuffer buffer = ByteBuffer.allocate(8);
+        NodeMetadata fromByteBuffer = NodeMetadata.fromByteBuffer(buffer);
+        assertNull(fromByteBuffer);
+    }
+}
```

Comment:
Please add a line break at the end of the file, it can be done in IDEA settings -> Editor -> General -> Ensure every saved file ends with a line break.